approval_polls/static/approval_polls/embed_instructions.js

```diff
@@ -0,0 +1,26 @@
+/* Code related to templates/approval_polls/embed_instructions.html */
```

Comment:
Wrap the entire file in an [IIFE](http://en.wikipedia.org/wiki/Immediately-invoked_function_expression)